starknet_py/tests/e2e/contract_interaction/v1_interaction_test.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from starknet_py.cairo.felt import decode_shortstring, encode_shortstring
+from starknet_py.tests.e2e.fixtures.constants import MAX_FEE
+from starknet_py.tests.e2e.fixtures.contracts import deploy_v1_contract
+
+
+@pytest.mark.asyncio
+async def test_general_v1_interaction(gateway_account):
```

Comment:
A concurrent PR with replacing gateway with full node in tests exists, so it's something that probably would have to be taken care of before merging into development (for now, in the tracking branch, it does not matter really)